lib/collection/script.js

```diff
@@ -62,6 +63,9 @@ _.assign(Script.prototype, /** @lends Script.prototype */ {
          * @type {string}
          */
         this.type = options.type || 'text/javascript';
+
+        this.packages = options.packages || {};
```

Comment:
Let's keep the `@type` properties for packages. Needs to be done for line 52 as well.